app/src/main/java/ch/hikemate/app/model/route/HikeRoute.kt

```diff
@@ -77,7 +81,39 @@ fun Bounds.toBoundingBox(): BoundingBox {
 }
 
 /** A simple data class to represent a latitude and longitude */
-data class LatLong(val lat: Double, val lon: Double)
+data class LatLong(val lat: Double, val lon: Double) {
+  /**
+   * Calculate the distance between this point and another point using the Haversine formula.
```

Comment:
Not sure what you're expecting? Do you want me to add a comment like "We use the Haversine formula because we use latitude and longitude coordinates and couldn't use Pythagore formulas"?

Reply:
Yes basically, `Calculate the distance between this point and another point, using the Haversine formula to account for earth's curvature.`

Reply:
Changed it!